.gitignore

```diff
@@ -4,4 +4,5 @@ data_ingestion/__pycache__
 jenniebrown.tfvars
 terraform.tfstate
 terraform.tfstate.backup
-upload_to_gcs.zip
\ No newline at end of file
+upload_to_gcs.zip
+venv
```

Comment:
nit/personal preference: I like leaving a blank line at the end so future diffs don't touch existing lines. Same with other files